packages/lesswrong/lib/collections/tags/views.ts

```diff
@@ -21,6 +22,7 @@ Tags.addView('tagBySlug', terms => {
   return {
     selector: {
       slug: terms.slug,
+      adminOnly: null
```

Comment:
Pretty sure this should use `viewFieldAllowAny` as you can for example see in the question views